tests/e2e/features/3DViewer.feature

```diff
@@ -0,0 +1,42 @@
+Feature: preview 3D model
+    As a user
+    I want to preview a 3D model
+    So that I can make sure this is the correct file (e.g. before sharing it with others)
+
+
+    Scenario: preview 3D model
+        Given the user has logged in with username "admin" and password "admin"
+        And the user has uploaded the following 3D models:
+            | model1.glb |
+            | model2.glb |
+
+        # preview 3D model
+        When the user previews the file "model1.glb" in the 3D model viewer
+        Then the 3D model "model1.glb" should be displayed in the viewport
+        And the file name "model1.glb" should be shown in the topbar
+
+        # toggle between standard and fullscreen mode
+        When the user selects "Enter full screen mode"
+        # should we use text that is on labels at all or just describe the action?
+        # reasoning: label text might change or be different if the UI is set to a different language...
+        Then the 3D model should be displayed in full screen mode
+        And the topbar should not be visible
+        When the user selects "Exit full screen mode"
```

Comment:
```suggestion
        When the user exits the full-screen mode
```
later if we add keyword navigation then we can have step something like this
`When the user exits the full-screen mode using "ESC" key`

Reply:
It seems like this is already implemented (maybe some default from ocis?), however it only works after clicking into the viewport...

Reply:
>It seems like this is already implemented (maybe some default from ocis?)

I think it's browser feat.

>however it only works after clicking into the viewport...

It should be fixed once we implement it (but can't say as of now)